
Training...







































Epoch: 62 | Loss: 0.0633:  97%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████▏   | 62/64 [01:18<00:02,  1.25s/it]

Epoch: 64 | Loss: 0.0625: 100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 64/64 [01:20<00:00,  1.26s/it]
Sampling...
Sampling Finished
128it [00:00, 696.56it/s]
Traceback (most recent call last):
  File "/home/l11/Documents/ProbabilisticMachineLearningProject/scripts/run_diffusion.py", line 121, in <module>
    main()
  File "/home/l11/Documents/ProbabilisticMachineLearningProject/scripts/run_diffusion.py", line 105, in main
    original_data, mask = inpainting_data.generate_data()
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/l11/Documents/ProbabilisticMachineLearningProject/src/utils.py", line 59, in generate_data
    mask = mask.reshape(1, -1)
           ^^^^
UnboundLocalError: cannot access local variable 'mask' where it is not associated with a value